pkgs/by-name/kc/kcc/package.nix

```
@@ -0,0 +1,63 @@
+{
+  stdenv,
+  lib,
+  qt6,
+  python3Packages,
+  fetchFromGitHub,
+  p7zip,
+  versionCheckHook,
+  archiveSupport ? true,
+  nix-update-script,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "kcc";
+  version = "7.3.3";
+
+  src = fetchFromGitHub {
+    owner = "ciromattia";
+    repo = "kcc";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-6zHUV4s1bOdARsTwNRxFM+s0p+6FLJhqJ9qG5BaBgas=";
+  };
+
+  nativeBuildInputs = [ qt6.wrapQtAppsHook ];
+
+  buildInputs = lib.optionals stdenv.hostPlatform.isLinux [ qt6.qtwayland ];
```

```suggestion
  buildInputs = lib.optionals stdenv.hostPlatform.isLinux [ qt6.qtwayland ] ++ [ qt6.qtbase ];
```
ofborg Darwin build fails without this